### DISCUSSION TITLE
Sort by fields, to sort by occurrences

### DISCUSSION THREAD

**Discussion by stanleykinkelaar** [spatie/flareapp.io-roadmap#95] [Ideas]
We would love to have a sort for the occurrences but it will likely be nice for other fields to.

**Comment by breadthe** [comment]
+1 for this.

At our company we just started a Flare trial, and overall I like it. But this is one thing I noticed right off the bat - a lack of sorting options.

At the minimum, I'd like to sort asc/desc by: datetime, occurrences, type, priority.
